### PR TITLE
Add more style, content and refactor js buttons repetitive code

### PR DESCRIPTION
**Pull request by CUPalex**
Update style.css.
Update index.css and questions array in script.js with some content. 
Refactor repetitive code for buttons.onclick in script.js into one function

### CODE REVIEW

**Review by PredaMihaiDragos — CHANGES_REQUESTED** [review]
I think you should add new lines at the end of both script.js and style.css files

File: portfolio/src/main/webapp/script.js [hunk 1]
```diff
@@ -106,23 +90,27 @@ function start() {
...
+        return;
+    }
     scoreElement.innerHTML = "Current score: " + score;
     question.innerHTML = questions[currentQuestion];
     leftButton.innerHTML = leftAnswers[currentQuestion];
     rightButton.innerHTML = rightAnswers[currentQuestion];
 }
+
+function endGame(score, questions, quizContainer) {
+    let endText = document.createElement("div");
```
Comment (PredaMihaiDragos) [inline portfolio/src/main/webapp/script.js]: const endText may be better, given that you never reasign it

**Comment by PredaMihaiDragos** [comment]
There's a comment ("adds a random greeting to the page") in script.js between lines 15-17, I think you forgot to delete it when you deleted a function.

**Comment by PredaMihaiDragos** [comment]
Mixing camelCase and dash-naming in HTML elements' class names is probably not a good idea for readability.

According to [Google style code for HTML and CSS](https://google.github.io/styleguide/htmlcssguide.html), you should only use dash-naming (4.1.10 section)

**Comment by PredaMihaiDragos** [comment]
AFAIK, it's a good practice to use [HTML Semantic elements](https://www.w3schools.com/html/html5_semantic_elements.asp), instead of `<div>` everywhere, for a lot of [reasons](https://stackoverflow.com/questions/17272019/why-use-an-html5-semantic-tag-instead-of-div). You can replace some of them with `<header> <nav> <section> <footer> etc`

File: portfolio/src/main/webapp/script.js [hunk 2]
```diff
@@ -106,23 +90,27 @@ function start() {
...
-        question.innerHTML = questions[currentQuestion];
-        leftButton.innerHTML = leftAnswers[currentQuestion];
-        rightButton.innerHTML = rightAnswers[currentQuestion];
+        nextQuestion(currentQuestion, questions, leftAnswers, rightAnswers, score, scoreElement, leftButton, rightButton, question, quizContainer);
     });
+    nextQuestion(currentQuestion, questions, leftAnswers, rightAnswers, score, scoreElement, leftButton, rightButton, question, quizContainer);
+}
 
+function nextQuestion(currentQuestion, questions, leftAnswers, rightAnswers, score, scoreElement, leftButton, rightButton, question, quizContainer) {
+    if (currentQuestion >= questions.length) {
```
Comment (martemyev) [inline portfolio/src/main/webapp/script.js]: What is the case when currentQuestion > questions.length?

File: portfolio/src/main/webapp/script.js [hunk 3]
```diff
@@ -106,23 +90,27 @@ function start() {
...
-            quizContainer.append(endText);
-            return;
-        }
-        scoreElement.innerHTML = "Current score: " + score;
-        question.innerHTML = questions[currentQuestion];
-        leftButton.innerHTML = leftAnswers[currentQuestion];
-        rightButton.innerHTML = rightAnswers[currentQuestion];
+        nextQuestion(currentQuestion, questions, leftAnswers, rightAnswers, score, scoreElement, leftButton, rightButton, question, quizContainer);
     });
+    nextQuestion(currentQuestion, questions, leftAnswers, rightAnswers, score, scoreElement, leftButton, rightButton, question, quizContainer);
```
Comment (martemyev) [inline portfolio/src/main/webapp/script.js]: I thought the next question appeared after a right or left button got clicked. What does this invocation do?

**Review by martemyev — CHANGES_REQUESTED** [review]

File: portfolio/src/main/webapp/script.js (hunk 1 above)
Comment (CUPalex) [reply portfolio/src/main/webapp/script.js]: Okay, thank you! Done.

File: portfolio/src/main/webapp/script.js (hunk 2 above)
Comment (CUPalex) [reply portfolio/src/main/webapp/script.js]: Oh, yes, there is no such case. Changed it to ==

File: portfolio/src/main/webapp/script.js (hunk 3 above)
Comment (CUPalex) [reply portfolio/src/main/webapp/script.js]: I changed the name of the function to clarify it. Here it is called to display the first question.

**Review by saiavin — APPROVED** [review]

**Review by PredaMihaiDragos — APPROVED** [review]

File: portfolio/src/main/webapp/script.js [hunk 4]
```diff
@@ -106,23 +86,32 @@ function start() {
...
-        rightButton.innerHTML = rightAnswers[currentQuestion];
+        displayQuestion(currentQuestion, questions, leftAnswers, rightAnswers, score, scoreElement, leftButton, rightButton, question, quizContainer);
     });
+    // display first question
+    displayQuestion(currentQuestion, questions, leftAnswers, rightAnswers, score, scoreElement, leftButton, rightButton, question, quizContainer);
+}
 
+function displayQuestion(currentQuestion, questions, leftAnswers, rightAnswers, score, scoreElement, leftButton, rightButton, question, quizContainer) {
+    // if it was the last question
+    if (currentQuestion == questions.length) {
```
Comment (martemyev) [inline portfolio/src/main/webapp/script.js]: I heard it's better to use '===' for comparison. Please see https://stackoverflow.com/q/359494, for example.

**Review by martemyev — APPROVED** [review]

File: portfolio/src/main/webapp/script.js (hunk 4 above)
Comment (CUPalex) [reply portfolio/src/main/webapp/script.js]: Yeah, it seems like it is always better to use '===' if applicable. Changed it!

**Pull request merged**